packages/create-cloudflare/templates/react-router/c3.ts

```diff
@@ -0,0 +1,52 @@
+import { logRaw } from "@cloudflare/cli";
+import { runFrameworkGenerator } from "frameworks/index";
+import { detectPackageManager } from "helpers/packageManagers";
+import type { TemplateConfig } from "../../src/templates";
+import type { C3Context } from "types";
+
+const { npm } = detectPackageManager();
+
+const generate = async (ctx: C3Context) => {
+	await runFrameworkGenerator(ctx, [
+		ctx.project.name,
+		"--template",
+		"https://github.com/remix-run/react-router-templates/tree/main/cloudflare",
+		// to prevent asking about git twice, just let c3 do it
+		"--no-git-init",
```

Comment:
Should we also set `--no-install` and let C3 handle that?

```suggestion
		"--no-git-init",
		"--no-install"
```